crates/uv/src/commands/project/init.rs

```diff
@@ -670,21 +695,34 @@ impl InitProjectKind {
         requires_python: &RequiresPython,
         python_request: Option<&PythonRequest>,
         vcs: Option<VersionControlSystem>,
+        no_authors: bool,
         no_readme: bool,
         package: bool,
     ) -> Result<()> {
         if !package {
             return Err(anyhow!("Library projects must be packaged"));
         }
 
+        fs_err::create_dir_all(path)?;
+
+        // Get the author information from git configuration.
+        let author = if no_authors {
+            Author::default()
+        } else {
+            get_author_from_git(path)
```

Comment:
What if the VCS isn't Git?

Reply:
We could just use Git as a source of information (a common one), it is valid information even if current VCS is not Git.

Reply:
Is it worthwhile to add a new dependency like [whoami](https://crates.io/crates/whoami) or [users](https://crates.io/crates/users) to fetch the OS username?